### ISSUE TITLE
Won't Return Some Legislators By State

### ISSUE THREAD

**Issue by rhythmsection** [sunlightlabs/congress#496]
Hey!

I'm having trouble getting the API to return all of the legislators by state--so far I've only been able to track the very easily apparent missing senators (so far, California's missing both Feinstein and Boxer, Florida's missing Nelson, NY is missing Gillibrand and TX Cornyn), but I think that some of the reps are missing from a state call too. I've been calling by https://congress.api.sunlightfoundation.com/legislators?state=[state abbr. ex. CA]&apikey=[apikey] and getting, oddly, most of the responses, but not all. Took a look at the data as being submitted, and all of the above listings have their state abbr. listed--and none of them seem to be any different from the rest, so it's hard to parse why I'm not getting them. 

Thoughts would be appreciated! 

Best,
KM

**Comment by konklone** [comment]
Hi KM,

By default, you'll only get 20 per page. You can use the "per_page" and
"page" params to change that, or use "per_page=all" (only on /legislators)
to get all of them at once. Boxer's definitely in there.

-- Eric

On Mon, Mar 16, 2015 at 7:51 PM, K. McClure <notifications@github.com>
wrote:

> Hey!
>
> I'm having trouble getting the API to return all of the legislators by
> state--so far I've only been able to track the very easily apparent missing
> senators (so far, California's missing both Feinstein and Boxer, Florida's
> missing Nelson, NY is missing Gillibrand and TX Cornyn), but I think that
> some of the reps are missing from a state call too. I've been calling by
> https://congress.api.sunlightfoundation.com/legislators?state=[state
> abbr. ex. CA]&apikey=[apikey] and getting, oddly, most of the responses,
> but not all. Took a look at the data as being submitted, and all of the
> above listings have their state abbr. listed--and none of them seem to be
> any different from the rest, so it's hard to parse why I'm not getting
> them.
>
> Thoughts would be appreciated!
>
> Best,
> KM
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunlightlabs/congress/issues/496>.
>



-- 
konklone.com | @konklone <https://twitter.com/konklone>

**Comment by handlers** [comment]
@rhythmsection — as @konklone says, the responses are paginated. I'm going to close this, but please this thread and reopen if pagination doesn't solve your issue.
